<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shop</class>
 <widget class="QWidget" name="Shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно покупки-возврата</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,0">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>800</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="LE_addProd">
       <property name="placeholderText">
        <string>Позиция</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_addAmount">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_price">
       <property name="placeholderText">
        <string>Цена за шт./л.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_salePrice">
       <property name="placeholderText">
        <string>Цена продажи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_addToBD">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить в базу</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="LE_removeProd">
       <property name="placeholderText">
        <string>Позиция</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_removeAmount">
       <property name="placeholderText">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_removeFromBD">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить из базы</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="PB_getAllProducts">
     <property name="text">
      <string>Посмотреть товары в наличии</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
